src/Component/Time.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Time(props) {
+  const date =  new Date(props.time)
+  const hours = date.getHours()
+  const minutes = date.getMinutes()
+  return (
+    <td>{`${hours}:${minutes < 10 ? '0'+ minutes : minutes}`}</td>
```

Comment:
try to use - https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/padStart